docs/mkdocs/docs/tutorials/data_organisation.md

```diff
@@ -0,0 +1,233 @@
+* Storage Performance: faster/slower storage for different applications according to requirements and cost
+
+
+### Library
+
+ArcticDB can handle a large number of libraries per object store - up to 100,000 is perfectly reasonable.
+
+A library groups together a set of related data. Typically its primary purpose is to help catalog the data - well named/documented libraries help people find the data they need.
+
+### Symbol
+
+The symbol is the base unit of storage in ArcticDB. Each symbol is a DataFrame and has optimised indexing on the columns and the index.
+
+So the choice of how to use the (symbol, columns, index) is important to how the system will perform. It is also the full key to access a specific single data item from a DataFrame (including the version, when needed).
+
+### Versions
+
+Each symbol gets a new version every time it is modified. A consideration of how versions will be treated is a key design choice - in particular it will have a big impact how much storage is used. The most common choices are:
+
+* Keep only the latest versions. Delete older versions either with `prune_previous_version` or in a background process (see [Enterprise Features](#enterprise-features)).
+* Keep key versions only. Often implemented using [snapshots](snapshots.md).
+* Keep all versions. This can use large amounts of storage if the data modifications are frequent but is very powerful for fully archiving the data.
+
+
+## System Design Concerns
+
+The following topics are necessary concerns when running a system. To that extent the data organisation plan must bear them in mind.
+
+### Initial Data Population
+
+This is usually a one-off process to get a system up and running. Ideally the same scripts can be used for initial data population and regular updates, with different parameters.
+
+### Data Update Process
+
+Regular updates to the data are the norm for most systems. We must consider in our design
+
+* Delivery format (the format that the data source uses) vs storage format of the data. The trade-offs are
+  * storing data in a format close to delivery format makes it easier to reconcile with the source
+  * the delivery format is often not the best format for downstream processes
+* Frequency of data updates
+* Storage growth characteristics over time
+* System performance over time
+
+### Efficient Downstream Access
+
+Often the data is accessed by downstream users and systems much more frequently than it is updated. If that is the case, it is worth organising data to optimise downstream access performance.
+
+### Data Problems: Investigation and Remediation
+
+System problems are often caused by suspected bad data. It is worth thinking about how to investigate and remediate such problems. For example
+
+* Tooling to find and examine suspected bad data
+* Consider how to reconcile suspected bad data vs the original data source
+* Manual overrides to fix bad data to allow the system to run effectively while data is re-sourced
+

```

Comment:
is it worth adding a note about how the data versioning makes it easy to restore to the last known good state?